README.md

```diff
@@ -80,7 +80,7 @@ Building a SPIR-V module, assembling it, parsing it, and then disassembling it:
 
 ```rust
 extern crate rspirv;
-extern crate spirv_headers as spirv;
+extern crate spirv as spirv;
```

Comment:
No need to `as spirv` here.

Reply:
Ah yes. In true Rust 2018 fashion we don't need `extern crate` at all anymore.